src/main/java/org/rutebanken/tiamat/config/Messages.java

```diff
@@ -0,0 +1,44 @@
+package org.rutebanken.tiamat.config;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.context.support.MessageSourceAccessor;
+import org.springframework.stereotype.Component;
+
+/**
+ * Util class to retrieve localized messages.
+ */
+@Component
+public class Messages {
```

Comment:
I'm wondering if the name of this class should be a bit more specific. LocalizedMessageRetriever or similar?